ftp/1.5/modules/ROOT/pages/ftp-examples.adoc

```diff
@@ -12,6 +12,8 @@ Configure the *List* operation to list the files and folders in the configured d
 Configure the *Copy* and *Move* operations to copy or move files or directories on demand.
 * xref:ftp-on-new-file.adoc[Trigger a Flow for a Newly Created or Updated File] +
 Configure the *On New or Updated File* source to trigger a flow for a new created or modified file.
+* xref:ftp-dynamic-connection.adoc[Connect to FTP Servers Dynamically] +
+Configure the *FTP Connector* to connect dynamically to multiple servers.
```

Comment:
What do we mean by "the *FTP Connector*"?
Is this a UI label? If so, it needs a common noun following its name, if preceded by "the."
If this is the name of the Anypoint Connector, then it should be Anypoint Connector for FTP (FTP Connector), no bold and no "the".